ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/pages/court-building-video/court-building-video.component.html

```diff
@@ -1,8 +1,28 @@
+
 <div>
-    <h1 class="govuk-heading-xl" i18n="@@courtBuildingVideo_h1">
-      Court building video
-    </h1>
+  <h1 class="vh-hide-h1" i18n="@@courtBuildingVideo_h1">Court building video</h1>
+</div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <app-video-view [source]="videoSource" (loaded)="videoLoaded()"></app-video-view>
   </div>
-  <div>
-      <button class="govuk-button" (click)="continue()" i18n="@@journey_btn_continue" id="continue" type="button">Continue</button>
+</div>
+
+<div class="govuk-grid-row vh-top-30">
+  <div class="govuk-grid-column-full">
+    <button type="submit" class="govuk-button govuk-!-margin-right-1" id="continue" (click)="continue()" i18n="@@courtBuildingVideo_btn_journey_continue">
```

Comment:
Should this be `i18n="@@journey_btn_continue"` or is it different?

Reply:
You should not use the same ID in the elements, despite you have the same word to translate, the ID should be unique, otherwise it will not be translated. System will translate only first one. So we should use format as pageName_tag_number to make it unique for each element in the app.

Reply:
Hmm, checking the docs:
https://angular.io/guide/i18n#define-unique-custom-ids

The expectation I would have is that we get the same translation for each button with that tag. Meaning if we change it from "Continue" to "Proceed" or something like that it would hit all pages.

It'll definitely work your way and it might be the correct way to do it since you can then control the texts on each page. Since it's a reused id in other places though I'm worried it'll get inconsistent, I'm sure we'll have other places that need to be adjusted when we actually localise anyway so if you think this way's the way to go let's do that.